### PR TITLE
ROECCT-119- Amend 'interrupt screen' flow

### PR DESCRIPTION
**Pull request by krenjith-ch**
### JIRA link

https://companieshouse.atlassian.net/browse/ROECCT-119

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: views/update/relevant-period-interrupt-card.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                <li>beneficiaries of a trust involved in the entity</li>
+            </ul>
+            <p class="govuk-body">
+                This is a new legal requirement so that we can understand who was involved in the entities that owned UK property or land when the Economic Crime and Transparency Bill came into force (28 February 2022).
+            </p>
+
+            <div class="govuk-inset-text">
+                <p class="govuk-body">
+                    The pre-registration period for <strong>{{ entity_name }}</strong> is between <strong>{{ start_date }}</strong> and <strong>{{ registration_date }}.</strong>
+                </p> </div>
```
Comment (ConnorOS-ch) [inline views/update/relevant-period-interrupt-card.html:39]: I still don't see why we can't do:

```
<div class="govuk-inset-text">
       {% include "includes/page-templates/relevant-period.html" %}
</div>
```
Should work, and I think it's much neater

File: src/config/index.ts [hunk 2]
```diff
@@ -307,6 +308,7 @@ export const UPDATE_ANY_TRUSTS_INVOLVED_URL = UPDATE_AN_OVERSEAS_ENTITY_URL + UP
 export const UPDATE_STATEMENT_VALIDATION_ERRORS_URL = UPDATE_AN_OVERSEAS_ENTITY_URL + UPDATE_STATEMENT_VALIDATION_ERRORS_PAGE;
 export const UPDATE_PAYMENT_FAILED_URL = UPDATE_AN_OVERSEAS_ENTITY_URL + PAYMENT_FAILED_PAGE;
 export const RELEVANT_PERIOD_OWNED_LAND_FILTER_URL = UPDATE_AN_OVERSEAS_ENTITY_URL + RELEVANT_PERIOD_OWNED_LAND_FILTER_PAGE;
+export const UPDATE_RELEVANT_PERIOD_INTERRUPT_URL = UPDATE_AN_OVERSEAS_ENTITY_URL + RELEVANT_PERIOD_INTERRUPT_PAGE;
```
Comment (ConnorOS-ch) [inline src/config/index.ts:311]: We need to be clear about these naming conventions, I was intending to go with just `RELEVANT_PERIOD_INTERRUPT_URL` not sure if we need both prefixes

File: views/update/relevant-period-interrupt-card.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
+{% extends "update-layout.html" %}
+{% import "includes/date-macros.html" as dateMacros %}
+{% set start_date = "28 February 2022" %}
+{% set registration_date = dateMacros.formatDate(dateOfCreation["day"], dateOfCreation["month"], dateOfCreation["year"]) %}
```
Comment (ConnorOS-ch) [inline views/update/relevant-period-interrupt-card.html:4]: Shouldn't need these dateMacros, start date, or registration date. It's all handled inside `relevant-period.html` now

**Review by ConnorOS-ch — CHANGES_REQUESTED** [review]
Nice work, I added a couple of comments

File: views/update/relevant-period-interrupt-card.html (hunk 1 above)
Comment (krenjith-ch) [reply views/update/relevant-period-interrupt-card.html]: change updated as suggested

File: src/config/index.ts (hunk 2 above)
Comment (krenjith-ch) [reply src/config/index.ts]: Updated

File: views/update/relevant-period-interrupt-card.html (hunk 3 above)
Comment (krenjith-ch) [reply views/update/relevant-period-interrupt-card.html]: changed as suggested

**Review by ConnorOS-ch — APPROVED** [review]
Looks good to me

**Review by amalviyach — APPROVED** [review]

**Pull request merged**